src/main/java/model/PreferencesModel.java

```diff
@@ -1,8 +1,98 @@
 package model;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 import observe.Observable;
 import observe.PreferencesObserver;
 
 public class PreferencesModel extends Observable<PreferencesObserver> {
 
+    public boolean musicEnabled;
```

Comment:
Oh well, OOP asks us to make all the fields private and add tons of get/set boilerplate, so let's do that here.